src/couch_replicator/src/couch_replicator_scheduler.erl

```diff
@@ -178,13 +178,13 @@ health_threshold() ->
         ?DEFAULT_HEALTH_THRESHOLD_SEC
     ).
 
--spec find_jobs_by_dbname(binary()) -> list(#rep{}).
+-spec find_jobs_by_dbname(binary()) -> list(rep_id()).
```

Comment:
The typespec is wrong here as we return a a list of rep_ids not `#rep{}` records. This is probably where the mocked test function in the doc_processor module had mistakenly copied the wrong type from.